gui/main_window.py

```diff
@@ -160,7 +160,8 @@ def log_in(self, user_account: UserAccount):
         LoginView().log_in(user_account)
         SplashScreen().wait_until_appears().wait_until_hidden()
         if not configs.DEV_BUILD:
-            BetaConsentPopup().wait_until_appears().confirm()
+            if driver.waitFor(lambda: BetaConsentPopup().exists, configs.timeouts.UI_LOAD_TIMEOUT_MSEC):
```

Comment:
Probably we need to add the same to sign_up()?

Reply:
added